ui/app/components/ui/circle-icon/circle-icon.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import CircleIcon from './circle-icon.component'
+import img from '../../../../../app/images/eth_logo.svg'
+
+storiesOf('CircleIcon', module)
```

Comment:
Both stories should be updated to use the new Component Story Format, as shown in #8108

Reply:
Done in fb625e1cf